docs/guides/pagination/offset-based.md

```diff
@@ -0,0 +1,394 @@
+---
+title: Pagination - Offset based
+---
+
+# Implementing Offset Pagination
+
+Make sure to check our introduction to pagination [here](./overview.md)!
+
+Let us implement offset based pagination in GraphQL. By the end of this tutorial, we
+should be able to return a list of users which can be paginated by the client.
```

Comment:
> which can be paginated by the client

Is this correct? Isn't the pagination technically done by the server?

Reply:
I modified the intro! Thanks!